App.tsx

```diff
@@ -275,6 +277,21 @@ export const App: React.FC<IProps> = (): React.ReactElement => {
     },
   ];
 
+  const TabItems = [
+    {
+      label: 'Tab1',
+      value: 0,
+    },
+    {
+      label: 'Tab2',
+      value: 1,
+    },
+    {
+      label: 'Tab3',
+      value: 2,
+    },
+  ];
+
```

Comment:
I know this was not introduced by this PR, but the url in line #296 seems wrong to me. 
Any ideas @ajaybushel @mdjacobs @bittu1028?
I would think we do not want to hard code an external URL and if we have to have this probably refer to a local image.